lms/static/sass/discussion/_discussion.scss

```diff
@@ -19,7 +19,7 @@ body.discussion {
 
     .post-cancel {
       @include float(left);
-      margin: ($baseline/2) 0 0 ($baseline*0.75);
+      @include margin(($baseline/2), 0, 0, ($baseline*0.75));
```

Comment:
nit: our sass styleguide says to use spaces around operators, eg `($baseline / 2)`